sbncode/LArG4/CMakeLists.txt

```
@@ -31,6 +31,20 @@ cet_build_plugin(MergeSimSourcesSBN art::EDProducer
   BASENAME_ONLY
 )
 
+cet_build_plugin(SimpleMerge art::EDProducer
+  LIBRARIES
+  lardataobj::MCBase
+  larcorealg::headers
+  art::Framework_Principal
+  art::Framework_Services_Registry
+  art::Persistency_Common
+  messagefacility::MF_MessageLogger
+  canvas::canvas
+  fhiclcpp::types
+  fhiclcpp::fhiclcpp
```

Almost all of these are already included when ~`art::module`~ `art::EDProducer` is used above:
```suggestion
  messagefacility::MF_MessageLogger
```